tests/fv3_conf/regional_rrfs_a_run.IN

```diff
@@ -0,0 +1,25 @@
+rsync -arv @[INPUTDATA_ROOT]/FV3_regional_input_data/. .
+
+rm -rf INPUT RESTART
+mkdir  INPUT RESTART
+
+rsync -arv @[INPUTDATA_ROOT]/fv3_regional_rrfs_a/INPUT/. INPUT/.
+
+cp @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/MERRA2/merra2.aerclim.2003-2014.m01.nc aeroclim.m01.nc
+cp @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/MERRA2/merra2.aerclim.2003-2014.m01.nc aeroclim.m02.nc
+cp @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/MERRA2/merra2.aerclim.2003-2014.m01.nc aeroclim.m03.nc
+cp @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/MERRA2/merra2.aerclim.2003-2014.m01.nc aeroclim.m04.nc
+cp @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/MERRA2/merra2.aerclim.2003-2014.m01.nc aeroclim.m05.nc
+cp @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/MERRA2/merra2.aerclim.2003-2014.m01.nc aeroclim.m06.nc
+cp @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/MERRA2/merra2.aerclim.2003-2014.m01.nc aeroclim.m07.nc
+cp @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/MERRA2/merra2.aerclim.2003-2014.m01.nc aeroclim.m08.nc
+cp @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/MERRA2/merra2.aerclim.2003-2014.m01.nc aeroclim.m09.nc
+cp @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/MERRA2/merra2.aerclim.2003-2014.m01.nc aeroclim.m10.nc
+cp @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/MERRA2/merra2.aerclim.2003-2014.m01.nc aeroclim.m11.nc
+cp @[INPUTDATA_ROOT]/FV3_input_data_INCCN_aeroclim/MERRA2/merra2.aerclim.2003-2014.m01.nc aeroclim.m12.nc
```

Comment:
Is this correct? Same file copied 12 times into 12 different files.

Reply:
Corrected